.github/workflows/sast.yml

```
@@ -56,3 +56,5 @@ jobs:
       uses: github/codeql-action/analyze@v3
       with:
         category: "/language:${{matrix.language}}"
+        disable-default-queries: true
```

actionlint (1.7.4)

input "disable-default-queries" is not defined in action "github/codeql-action/analyze@v3". available inputs are "add-snippets", "category", "check_name", "checkout_path", "cleanup-level", "expect-error", "matrix", "output", "ram", "ref", "sha", "skip-queries", "threads", "token", "upload", "upload-database", "wait-for-processing"

(action)